### DISCUSSION TITLE
在想基于 web3:// 哪些应用场景是当前最需要的或频率最高的？

### DISCUSSION THREAD

**Discussion by luffythink** [IntensiveCoLearning/Web3-URL#118] [General]
并不是所有的东西都需要去中心化，去中心和中心化如何平衡？🤔

**Comment by qzhodl** [comment]
这是个好问题，我个人觉得至少现在Defi的前端需要完全的去中心化存储，并且使用类似之前智能合约升级的治理流程来管理版本

**Comment by qizhou** [comment]
> 并不是所有的东西都需要去中心化，去中心和中心化如何平衡？🤔

的确我们看到大量的应用，甚至很多公链都在往中心化的方向狂奔，对于去中心化的坚持越来越少。

除了DeFi之外，我想到的另外一个例子是Tornado cash的开发，现在github是archive模式，所以无法进行任何的修改。去中心化存储和web3://可以很好的解决这个问题。设置未来以太坊是不是可以通过web3://上面的git来开发，似的我们可以在以太坊（主网）上面开发以太坊（代码），也是一个有意思的话题。
